uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/leave-request-api.js

```diff
@@ -26,6 +26,7 @@ define([
       all: function (filters, pagination, sort, params, cache) {
         $log.debug('LeaveRequestAPI.all');
 
+        sort = sort || 'from_date DESC';
```

Comment:
controller can still pass it and override the default behaviour here otherwise same will be used at multiple places